CMakeLists.txt

```diff
@@ -6,13 +6,17 @@ project(picoprobe)
 
 pico_sdk_init()
 
+# Comment out to use Pico LED
+add_definitions(-DUSE_WS2812)
+
 add_executable(picoprobe
         src/led.c
         src/main.c
         src/usb_descriptors.c
         src/probe.c
         src/cdc_uart.c
         src/get_serial.c
+        src/ws2812.c
```

Comment:
Does it make sense to make the compilation of this (and also `ws2812.pio`) conditional on the value of `USE_WS2812` ? :shrug:

Reply:
Will do.